src/containers/MainScreen/index.js

```diff
@@ -23,12 +36,19 @@ const MainScreen = () => {
   );
 };
 
-MainScreen.options = {
-  topBar: {
-    title: {
-      text: translate('MAIN_SCREEN.title')
-    }
+MainScreen.navigationOptions = {
+  title: translate('MAIN_SCREEN.title'),
+  headerStyle: {
```

Comment:
If i recall you can define these styles when you define the stack, maybe it's better to all this in the stack definition? WDYT?

Reply:
Yeah, if there are global stacks, we should use the `defaultNavigationOptions` on the stack itself. In this case I just wanted to change a bit the styles of the main screen to leave an example on how to pass `navigationOptions` to a specific screen